e2e/cypress/support/compliance_export_commands.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+Cypress.Commands.add('enableComplianceExport', (exportFormate = 'csv') => {
```

Comment:
Please rename, move to `e2e/cypress/support/ui/compliance_export.js` and add corresponding entry to `e2e/cypress/support/ui/compliance_export.d.ts`
```suggestion
Cypress.Commands.add('uiEnableComplianceExport', (exportFormat = 'csv') => {
```